src/main/scala/com.snowplowanalytics.iglu/ctl/Storage.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2012-2019 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.iglu.ctl
+
+import java.util.concurrent.{ExecutorService, Executors}
+
+import scala.concurrent.ExecutionContext
+
+import cats.effect._
+
+import doobie.implicits._
+import doobie.util.transactor.Transactor
+
+import com.snowplowanalytics.iglu.ctl.Command.DbConfig
+
+case class Storage[F[_]](xa: Transactor[F]) {
+
+  /**
+    * Fetch columns of the given database table
+    */
+  def getColumns(tableName: String, tableSchema: String)(implicit F: Bracket[F, Throwable]): F[List[Storage.Column]] = {
+    (fr"SELECT column_name FROM information_schema.columns" ++
```

Comment:
Hey @aldemirenes, is `information_schema.columns` something Postgres-specific?

Reply:
I think you asked me and I approved it, but I had something completely different on my mind back then.

Reply:
It is Postgres-specific however it is working with Redshift too. I made some tests and it seems that it is working fine. Do you think there is a problem with using it ?